async_fuse/src/aligned_bytes.rs

```diff
@@ -1,6 +1,3 @@
-#![deny(missing_debug_implementations)]
-#![deny(missing_docs)]
-
 //! Inline crate from <https://crates.io/crates/aligned-bytes/0.1.1>
```

Comment:
aligned-bytes v0.1.1 has been yanked because of unsoundness.
Consider using v0.1.4.

See <https://crates.io/crates/aligned-bytes>

Reply:
How about check in AlignedBytes code into DatenLord?